free-podcasts-screencasts-en.md

```diff
@@ -125,6 +125,8 @@
 
 ### Haskell
 
+* [Haskell Tutorial](https://www.youtube.com/watch?v=02_H3LjqMr8) (screencast)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator.

Reply:
It is a single video so it is not in any playlist,
but I can add the producer name.

Should I keep the link even it is not a playlist?

Reply:
its ok